pages/icerpc/slic-transport/connection-idle-timeout.md

```diff
@@ -5,28 +5,37 @@ description: Understand connection idle timeout.
 
 ## Idle Timeout
 
-The transport implementation considers the connection lost if it doesn't receive data within the period defined by the
+The transport implementation considers the connection lost if it doesn't receive any data within the period defined by
 idle timeout. The idle timeout is specified with the [IdleTimeout][connection-parameters] parameter. Each side of the
 connection advertises its idle timeout and the minimum value is used for the connection.
 
+{% callout %}
+The default idle timeout is 30 seconds.
+{% /callout %}
+
 ## Keeping a connection alive
 
-To prevent the idle timeout from being triggered, the client or server can send a [Ping] frame. When the peer receives
-this `Ping` frame, it resets its idle timeout timer and send back a [Pong] frame. The reception of the `Pong` frame will
-in turn reset the idle timeout timer of the receiver.
+Slic is responsible to keep a connection from becoming idle when the application doesn't write anything to this
+connection.
 
-It's up to the transport implementation to decide if and when `Ping` frames are sent.
+{% callout %}
+TODO: this behavior should be configurable and controlled by the upper protocol (e.g., `icerpc`). Currently, it isn't.
 
-A `Ping` frame carries an opaque payload. This payload is sent back with the `Pong` frame. Multiple `Ping` frames can be
-sent concurrently.
+See also [InactivityTimeout].
```

Comment:
I don't see the point of the TODO and the reference to the `icerpc` protocol inactivity timeout property.

Reply:
Removed TODO and reference to Inactivity Timeout.